### PR TITLE
Add a `tower.yml` file to enable Reports in Nextflow Tower

### PR DESCRIPTION
**Pull request by ewels**
Untested in Tower so far.

### CODE REVIEW

**Review by FelixKrueger — APPROVED** [review]
Will test straight away

File: tower.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+reports:
+  multiqc_report.html:
+    display: "MultiQC HTML report"
+  bismark_summary_report.html:
+    display: "Bismark summary report"
+  "**/bismark/reports/*/*report.html":
+    display: "All samples Bismark reports"
+  "**/qualimap/*/qualimapReport.html":
+    display: "All samples Qualimap reports"
+  "**/fastqc/*_fastq.html":
```
Comment (FelixKrueger) [inline tower.yml:10]: Maybe try to change this to

```
"**/fastqc/*_fastqc.html":
```

**Review by FelixKrueger — CHANGES_REQUESTED** [review]

File: tower.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+reports:
+  multiqc_report.html:
+    display: "MultiQC HTML report"
+  bismark_summary_report.html:
+    display: "Bismark summary report"
+  "**/bismark/reports/*/*report.html":
+    display: "All samples Bismark reports"
+  "**/qualimap/*/qualimapReport.html":
```
Comment (ewels) [inline tower.yml:8]:
```suggestion
  qualimapReport.html:
```

File: tower.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    display: "All samples Qualimap reports"
+  "**/fastqc/*_fastqc.html":
```
Comment (ewels) [inline tower.yml:10]:
```suggestion
  "*_fastqc.html":
```

File: tower.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+reports:
+  multiqc_report.html:
+    display: "MultiQC HTML report"
+  bismark_summary_report.html:
+    display: "Bismark summary report"
+  "**/bismark/reports/*/*report.html":
```
Comment (ewels) [inline tower.yml:6]:
```suggestion
  "bismark/reports/*/*report.html":
```

File: tower.yml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+reports:
+  multiqc_report.html:
+    display: "MultiQC HTML report"
```
Comment (ewels) [inline tower.yml:3]:
```suggestion
    display: "MultiQC report"
```

File: tower.yml [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+  bismark_summary_report.html:
+    display: "Bismark summary report"
+  "bismark/reports/*/*report.html":
+    display: "All samples Bismark reports"
```
Comment (ewels) [inline tower.yml:7]:
```suggestion
    display: "Bismark sample report"
```

File: tower.yml [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+  qualimapReport.html:
+    display: "All samples Qualimap reports"
```
Comment (ewels) [inline tower.yml:9]:
```suggestion
    display: "Qualimap sample report"
```

File: tower.yml [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+  "*_fastqc.html":
+    display: "All samples FastQC reports, before and after trimming"
```
Comment (ewels) [inline tower.yml:11]:
```suggestion
    display: "FastQC reports"
```

**Review by ewels — COMMENTED** [review]
Titles are a bit verbose in Tower..

File: tower.yml [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+  multiqc_report.html:
+    display: "MultiQC report"
+  bismark_summary_report.html:
+    display: "Bismark summary report"
+  "bismark/reports/*/*report.html":
+    display: "Bismark sample report"
+  qualimapReport.html:
+    display: "Qualimap sample report"
+  "*_fastqc.html":
+    display: "FastQC reports"
```
Comment (ewels) [inline tower.yml:11]:
```suggestion
  "*_fastqc.html":
    display: "FastQC report, before trimming"
  "*_val_*_fastqc.html":
    display: "FastQC report, after trimming"
```

File: tower.yml [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+  bismark_summary_report.html:
+    display: "Bismark summary report"
+  "bismark/reports/*/*report.html":
+    display: "Bismark sample report"
+  qualimapReport.html:
+    display: "Qualimap sample report"
+  "*_fastqc.html":
+    display: "FastQC report, before trimming"
+  "*_val_*_fastqc.html":
+    display: "FastQC report, after trimming"
```
Comment (ewels) [inline tower.yml:13]:
```suggestion
reports:
  multiqc_report.html:
    display: "MultiQC report"
  bismark_summary_report.html:
    display: "Bismark summary report"
  "bismark/reports/*report.html":
    display: "Bismark sample report"
  "*_val_*_fastqc.html":
    display: "FastQC report, after trimming"
  "*_fastqc.html":
    display: "FastQC report, before trimming"
```

**Comment by ewels** [comment]
Just failing linting because of the new tools release. Will sort that out with the template merge PR.

**Pull request merged**